src/rules/custom-property-empty-line-before/__tests__/index.js

```diff
@@ -0,0 +1,202 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+
+  accept: [ {
+    code: "a {\n\n --custom-prop: value;\n}",
+  }, {
+    code: "a {\r\n\r\n --custom-prop: value;\r\n}",
+  }, {
+    code: "a{\n\n --custom-prop: value; }",
+  }, {
+    code: "a{\n\n --custom-prop: value;\r\n\r\n --custom-prop2: value;}",
+  }, {
+    code: "a{\n\n --custom-prop: value;\n\r\n --custom-prop2: value;}",
+  }, {
+    code: "a{\n top: 10px;\n\r\n --custom-prop: value;}",
+  }, {
+    code: "a{\n @extends .class;\n\r\n --custom-prop: value;}",
+  }, {
+    code: "a{\n $var: value;\n\r\n --custom-prop: value;}",
+  } ],
+
+  reject: [ {
+    code: "a {\n --custom-prop: value;\n}",
```

Comment:
Can we change this to `a {\n--custom-prop: value;\n}` and the column to `1` so we can be sure the column index is being calculated properly as all the others are `2`.